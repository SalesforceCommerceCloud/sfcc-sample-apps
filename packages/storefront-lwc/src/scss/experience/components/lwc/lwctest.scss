.lwctest {
    color: chocolate;
    background-color: azure;
}
