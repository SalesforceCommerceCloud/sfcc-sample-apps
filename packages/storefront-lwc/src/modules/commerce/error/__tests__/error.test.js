import { createElement } from 'lwc';
import ErrorCmp from 'commerce/error';

describe('commerce/error', () => {
    beforeEach(() => {
        while (document.body.firstChild) {
            document.body.removeChild(document.body.firstChild);
        }
    });

    it('renders correctly', () => {
        const element = createElement('commerce-error', { is: ErrorCmp });
        document.body.appendChild(element);

        return Promise.resolve().then(() => {
            expect(element).toMatchSnapshot();
        });
    });
});
