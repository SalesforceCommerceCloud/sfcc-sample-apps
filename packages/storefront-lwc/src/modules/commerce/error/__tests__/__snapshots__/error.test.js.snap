// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`commerce/error renders correctly 1`] = `
<commerce-error>
  #shadow-root(open)
    Error
</commerce-error>
`;
